cv_bridge/include/cv_bridge/cv_bridge.h

```diff
@@ -93,6 +98,9 @@ class CvImage
    */
   void toImageMsg(sensor_msgs::Image& ros_image) const;
 
+  void toCompressedImageMsg(sensor_msgs::CompressedImage& ros_image, const std::string& dst_format = std::string()) const;
```

Comment:
sorry, I meant: explain as doxygen comments so that people using the function know how to use it.